### DISCUSSION TITLE
How do I fill a valuebox with OJS?

### DISCUSSION THREAD

**Discussion by h3ndrik** [quarto-dev/quarto-cli#8822] [Q&A] [answered]
### Description

Is there a way to fill the dasboard valuebox with `ojs`?

````qmd
---
title: "test"
format: dashboard
---

```{ojs}
value = 42
```

```{ojs}
//| content: valuebox
//| title: "Test"
//| icon: pencil
//| color: primary
???
```

````

Unfortunately the documentation doesn't include an example for ojs. And I can't get the dict right. I've had a look at the ojs documentation. And I was able to output a JS Object with a value key, but that's also not rendered like the numbers from R or python, but shows up as Object {value: 42}

**Comment by mcanouil** [comment]
Why don't you use "value"?

````qmd
---
title: "test"
format: dashboard
---

## Row 

```{ojs}
//| include: false
value = 42
```

```{ojs}
//| content: valuebox
//| title: "Test"
//| icon: pencil
//| color: primary
value
```
````

**Reply by mcanouil** [reply, marked as answer]
I opened an issue to track this:
- https://github.com/quarto-dev/quarto-cli/issues/8823

**Reply by mcanouil** [reply]
Workaround use the markdown syntax using fenced divs.

````qmd
---
title: "test"
format: dashboard
---

## Row 

```{ojs}
//| include: false
value = 12
```

::: {.valuebox icon="pencil" color="primary"}
Articles per day

`{ojs} value`
:::
````

**Reply by h3ndrik** [reply]
Thank you very much. That works fine.
